backend/canvas_app_explorer/lti1p3.py

```diff
@@ -158,10 +158,17 @@ def create_user_in_django(request: HttpRequest, message_launch: ExtendedDjangoMe
     user_obj.backend = 'django.contrib.auth.backends.ModelBackend'
     django_login(request, user_obj)
 
-    if course_id is not None and isinstance(course_id, int):
-        request.session['course_id'] = course_id
+    course_id_error = Exception('The canvas_course_id custom LTI variable must be configured.')
+    if course_id is not None:
```

Comment:
Not exactly sure this would ever come up in practice, it'd be more likely than we'd get `KeyError` with `course_id` being undefined. However, it is technically possible, since it is `None` if you set the value of the `course_id` key to `null` in the LTI config.